src/Endpoints/Indexes.php

```diff
@@ -153,6 +153,11 @@ public function delete(): array
         return $this->http->delete(self::PATH.'/'.$this->uid) ?? [];
     }
 
+    public function swapIndexes(array $indexes): array
```

Comment:
```suggestion
    /**
      * @param array{0: list<non-empty-string>, 1: list<non-empty-string>}
      */
    public function swapIndexes(array $indexes): array
```

Doc could help for phpstan :)

Reply:
Amazing, thanks @norkunas 🎉

Reply:
>     * @param array{0: list<non-empty-string>, 1: list<non-empty-string>} $indexes

Not sure i understand the syntax, is it a type working with the following example?

```
[
  { indexes: ['a', 'b'] },
  { indexes: ['a', 'b'] }
]
```

Reply:
was on my phone, probably misunderstood from the tests.
then it should be:
```
@param list<array{indexes: list<non-empty-string>}>
```